### ISSUE TITLE
Not able to merge a pull request

### ISSUE THREAD

**Issue by mohit84** [gluster/project-infrastructure#98]
As a maintainer i am not able to approve my pull request.The pull request is already reviewed by peer maintainers but
peer maintainer does not have a merging rights so current infra is not considering their votes to merge a pull request.It means
any pull request can be merge only if it is approved by other maintainer, the behavior is slightly different from earlier gerrit 
behavior. I have been trying to merge this https://github.com/gluster/glusterfs/pull/1666 and i am not able to do it.We need
to change this behavior specific to back-porting patches into gluster downstream releases.

**Comment by deepshikhaaa** [comment]
@amarts @mscherer Do you have any comments on this issue? I looked for an option where an author can approve its own PR but could not find any such setting.

**Comment by mscherer** [comment]
I do not think this can be done from what I understand in the github documentation, we can require a review and set the number of people who have to review: https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuring-protected-branches 

But there is no settings to bypass that (afaik).

**Comment by deepshikhaaa** [comment]
Or we can just remove the protection branch rule which will just remove the review required before merge!

Personally, I don't feel it is a good option.

**Comment by mscherer** [comment]
I also do not feel that's a good option. I am also not convinced that the decision of downstream to accept a patch mean that it would be safe to merge upstream, or at least, not more than any other patchs.
